lib/livebook_web/live/with_password_toggle_component.ex

```diff
@@ -0,0 +1,16 @@
+defmodule LivebookWeb.WithPasswordToggleComponent do
+  use LivebookWeb, :live_component
+
+  @impl Phoenix.LiveComponent
+  def render(assigns) do
+    ~H"""
+    <div id={"password-toggle-#{@id}"} class="flex input w-min items-center" phx-hook="PasswordToggle">
```

Comment:
We can make this component a bit more transparent by embedding the input as-is and adding the icon with absolute positioning :)

```suggestion
    <div id={"password-toggle-#{@id}"} class="relative w-min" phx-hook="PasswordToggle">
```